### ISSUE TITLE
RaiNews: Unable to extract relinker URL

### ISSUE THREAD

**Issue by rusemode** [yt-dlp/yt-dlp#3911]
### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/ytdl-org/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

Italy

### Example URLs

https://www.rainews.it/rubriche/24mm/video/2022/05/24mm-del-29052022-12cf645d-1ffd-4220-b27c-07c226dbdecf.html

### Description

Seems to be working the same way Raiplay does but when trying to use yt-dlp, it fails to extract relinker URL

### Verbose log

```shell
C:\ytdl>yt-dlp.exe -vU https://www.rainews.it/rubriche/24mm/video/2022/05/24mm-del-29052022-12cf645d-1ffd-4220-b27c-07c226dbdecf.html
[debug] Command-line config: ['-vU', 'https://www.rainews.it/rubriche/24mm/video/2022/05/24mm-del-29052022-12cf645d-1ffd-4220-b27c-07c226dbdecf.html']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d523] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19042-SP0
[debug] Checking exe version: ffprobe -bsfs
[debug] Checking exe version: ffmpeg -bsfs
[debug] exe versions: ffmpeg git-2020-08-31-4a11a6f, ffprobe git-2020-08-31-4a11a6f
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
Latest version: 2022.05.18, Current version: 2022.05.18
yt-dlp is up to date (2022.05.18)
[debug] [Rai] Extracting URL: https://www.rainews.it/rubriche/24mm/video/2022/05/24mm-del-29052022-12cf645d-1ffd-4220-b27c-07c226dbdecf.html
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading webpage
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading video JSON
ERROR: [Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Unable to extract relinker URL; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 642, in extract
  File "yt_dlp\extractor\rai.py", line 707, in _real_extract
  File "yt_dlp\extractor\common.py", line 1226, in _search_regex
```

**Comment by gamer191** [comment]
Rai news is actually already supported. So this is obviously a bug in the Rai News extractor

**Comment by someziggyman** [comment]
here's another test link for this case:
https://www.rainews.it/tgr/piemonte/notiziari/video/2022/07/TGR-Piemonte-del-17072022-ore-1400-0012d7a7-a5d5-43e6-87b9-6d6327e7247a.html

**Comment by pukkandan** [comment]
@nixxo

**Comment by nixxo** [comment]
will look into it.

**Comment by nixxo** [comment]
ok:

```
❯ ydpm https://www.rainews.it/rubriche/24mm/video/2022/05/24mm-del-29052022-12cf645d-1ffd-4220-b27c-07c226dbdecf.html -F
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading webpage
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading video JSON
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading XML metadata for platform mon
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading m3u8 information
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading XML metadata for platform flash
[Rai] 12cf645d-1ffd-4220-b27c-07c226dbdecf: Downloading XML metadata for platform native
[info] Available formats for 12cf645d-1ffd-4220-b27c-07c226dbdecf:
ID         EXT RESOLUTION FPS |   FILESIZE   TBR PROTO | VCODEC        VBR ACODEC    ABR
----------------------------------------------------------------------------------------
hls-1296   mp4 768x432     25 | ~251.47MiB 1296k m3u8  | avc1.4d401f 1296k mp4a.40.2  0k
https-1200 mp4 768x432     25 | ~251.47MiB 1296k https | avc1.4d401f 1296k mp4a.40.2  0k
```

will cleanup the code and make a test for it.
